web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -116808,6 +116818,65 @@
                 })
             }
         })),
+        InProgresstList = Object(f.f)(Object(T.f)((function(e) { // BAO
+            return {
+                _inProgressList: e["features/inprogress-list"]
+            }
+        }))(class extends Pa {
+            constructor(e) {
+                super(e), this._getRenderListEmptyComponent = this._getRenderListEmptyComponent.bind(this), this._onPress = this._onPress.bind(this)
+            }
+            componentDidMount() {
+                super.componentDidMount(),
+                this._importJson();
+                if (interfaceConfig.IN_PROGRESS_LIST_INTERVAL > 0) {
+                    this._interval = setInterval(() => this._importJson(), interfaceConfig.IN_PROGRESS_LIST_INTERVAL * 1000)
+                }
+            }
+            componentWillUnmount() {
+                this._interval && (clearInterval(this._interval), this._interval = void 0)
+            }
+            _importJson() {
+                fetch("inProgressList.json")
+                    .then(res => res.ok && res.json())
+                    .then(data => this.props.dispatch({type: "UPDATE_INPROGRESS_LIST", list: data}))
```

Comment:
After reading the JSON, the "UPDATE_INPROGRESS_LIST" event is dispatched.
The event is processed in line 119775.